package.json

```
@@ -24,7 +24,9 @@
         "typescript": "^4.5.5"
     },
     "scripts": {
-        "build": "tsc --build packages/* test",
+        "build": "npm run build:es && npm run build:cjs",
+        "build:es": "tsc --build packages/* test",
+        "build:cjs": "tsc -p packages/parse5/tsconfig.cjs.json && echo '{\"type\":\"commonjs\"}' > packages/parse5/dist/cjs/package.json",
```

```suggestion
        "build": "npm run build:esm && npm run build:cjs",
        "build:esm": "tsc --build packages/* test",
        "build:cjs": "tsc -p packages/parse5/tsconfig.cjs.json && echo '{\"type\":\"commonjs\"}' > packages/parse5/dist/cjs/package.json",
```

`esm` is used a bunch everywhere. And both 3 characters means it aligns nicely :)

---

Good recommendation! 😄